backend/SafeSender.StorageAPI/docker-compose.yml

```diff
@@ -8,22 +8,21 @@ services:
       - "27017:27017"
     volumes:
       - safe-sender-mongodb-data:/data/db
-    external_links:
-      - "mongo-data"
-  
+    restart: always
+
   safe-sender-storage-api:
     build:
       context: .
       dockerfile: SafeSender.StorageAPI/Dockerfile
     ports:
       - "5006:80"
-      - "5007:443"
-    external_links:
-      - "safe-sender-storage-api"
     environment:
-      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_ENVIRONMENT=${ASPNETCORE_ENVIRONMENT:-Production}
```

Comment:
In this configuration, the ASPNETCORE_ENVIRONMENT environment variable will be set to the value of the ASPNETCORE_ENVIRONMENT variable in your shell. If that variable isn't set, it will default to Production.